reliability-v2/README.md

```diff
@@ -17,30 +18,46 @@ cd svt/reliability-v2
 [tmux](https://github.com/tmux/tmux/wiki) is a tool that can help to keep the reliability running in the background to avoid the termination of the run due to the unexpected termination of the terminal. Ad reliability test sometimes run for several days, we recommend you to run reliability in a tmux session.
 
 ## Prepare Authentication files
-If you installed your cluster with [Flexy-install](https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/ocp-common/job/Flexy-install/), download kubeconfig, users.spec and kubeadmin-password files from the Build Artifacts after the job completed successfully. The user `kubeadmin` has admin priviledge and its password is in kubeadmin-password. users.spec contains username and password pairs of 50 test users.
+You need to prepare 3 files
+1. kubeconfig
+2. admin: a file contains pairs of username and password of user which is admin role. e.g. kubeadmin:password (No space)
+3. users: a file contains pairs of usernames and passwords. e.g testuser-0:password1,testuser-1:password2,...,testuser-n:passwordn (No space)
+Use either of the options below to generate the 3 files.
+
+Option 1
+[generate_auth_files.sh](https://github.com/openshift/svt/blob/master/reliability-v2/utils/generate_auth_files.sh) can help you to generate the required files mentioned above. Get the command's help with generate_auth_files.sh -h.
+
+Option 2
+If you installed your cluster with [Flexy-install](https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/ocp-common/job/Flexy-install/), download kubeconfig, users.spec and kubeadmin-password files from the Build Artifacts after the job completed successfully. The user `kubeadmin` has admin priviledge and its password is in kubeadmin-password. users.spec contains username and password pairs of 50 test users. Then compose the above 3 files based on the files you get.
+
+If you installed your rosa cluster with Prow CI, follow below instruction to get the kubeconfig, admin credential file and users credential files. Then compose the above 3 files based on the files you get.
 
-Replace `kubeconfig` `kubeadmin_password` and `user_file` with above files in your config file.
+1. Open the job , find the link following "Using namespace".
+2. Open the link you get above and login, click your name on right top of the console, choose "Copy login command". You may need to open the link in a new window in Chrome to get the token.
```

Comment:
@qiliRedHat Can you also add to try in "incognito" window if above does not work. 
When both Paige and I tried following this and tried to "Copy login command" and open in Chrome we got permission error.  But when you  "Copy login command" and open in incognito window, it works.

Reply:
I have mentioned this [before](https://github.com/openshift/svt/pull/762#discussion_r1235729255), but looks like it's missing. Looks good otherwise.

Reply:
@shahsahil264 I updated to use incognito window.